specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2024-07-01-preview/loadtestservice.json

```
@@ -0,0 +1,6572 @@
+      "name": "search",
+      "in": "query",
+      "description": "Prefix based, case sensitive search on searchable fields - displayName,\ncreatedBy. For example, to search for a test, with display name is Login Test,\nthe search parameter can be Login.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.executionFrom": {
+      "name": "executionFrom",
+      "in": "query",
+      "description": "Start DateTime(RFC 3339 literal format) of test-run execution time filter range.",
+      "required": false,
+      "type": "string",
+      "format": "date-time",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.executionTo": {
+      "name": "executionTo",
+      "in": "query",
+      "description": "End DateTime(RFC 3339 literal format) of test-run execution time filter range.",
+      "required": false,
+      "type": "string",
+      "format": "date-time",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.maxpagesize": {
+      "name": "maxpagesize",
+      "in": "query",
+      "description": "Number of results in response.",
+      "required": false,
+      "type": "integer",
+      "format": "int32",
+      "default": 50,
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.orderby": {
+      "name": "orderby",
+      "in": "query",
+      "description": "Sort on the supported fields in (field asc/desc) format. eg: executedDateTime\nasc. Supported fields - executedDateTime",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.search": {
+      "name": "search",
+      "in": "query",
+      "description": "Prefix based, case sensitive search on searchable fields - description,\nexecutedUser. For example, to search for a test run, with description 500 VUs,\nthe search parameter can be 500.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.status": {
+      "name": "status",
+      "in": "query",
+      "description": "Comma separated list of test run status.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTestRunQueryParams.testId": {
+      "name": "testId",
+      "in": "query",
+      "description": "Unique name of an existing load test.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTriggerQueryParams.createdFrom": {
+      "name": "createdFrom",
+      "in": "query",
+      "description": "Start DateTime of trigger creation time filter range. (RFC 3339 literal format)",
+      "required": false,
+      "type": "string",
+      "format": "date-time",
+      "x-ms-parameter-location": "method"
+    },
+    "ListTriggerQueryParams.createdTo": {
+      "name": "createdTo",
```

Another rename for consistency
```suggestion
      "name": "createdDateEndTime",
```